services/gitlab/gitlab-license.service.js

```diff
@@ -0,0 +1,73 @@
+import Joi from 'joi'
+import { renderLicenseBadge } from '../licenses.js'
+import GitLabBase from './gitlab-base.js'
+
+const schema = Joi.object({
+  license: Joi.object({
+    name: Joi.string().required(),
+  }).allow(null),
+}).required()
+
+const documentation = `
+<p>
+  You may use your GitLab Project Id (e.g. 13083) or your Project Path (e.g. gitlab-org/gitlab-foss )
+</p>
+`
+const commonProps = {
+  namedParams: {
+    project: 'gitlab-org/gitlab-foss',
+  },
+  documentation,
+}
+
+export default class GitlabLicense extends GitLabBase {
+  static category = 'license'
+
+  static route = {
+    base: 'gitlab/v/license',
+    pattern: ':project+',
+  }
+
+  static examples = [
+    {
+      title: 'GitLab',
+      ...commonProps,
+      staticPreview: {
+        label: 'license',
+        message: 'MIT',
+        color: 'green',
+      },
+    },
```

Comment:
Will probably need to add an example that includes a custom instance usage too

Reply:
done